onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -423,10 +421,6 @@ struct MLAS_PLATFORM {
     uint32_t NchwcBlockSize;
     uint32_t PreferredBufferAlignment;
 #endif
-
-#if defined(MLAS_USE_WIN32_THREADPOOL)
-    int32_t MaximumThreadCount;
```

Comment:
Can you also remove the now unneeded GetSystemInfo code from platform.cpp?

Reply:
Done.